app/controllers/Assemblies.java

```diff
@@ -1213,6 +1213,9 @@ public static Result uploadGroupUsers(@ApiParam(name = "aid", value = "Assembly
 									invitation.setModerator(false);
 								}
 
+								if(wg.getCreator() == null) {
+									return internalServerError("The working group doesn't have a creator");
```

Comment:
@loorely actually, the creator is always the user associated to the `sessionKey`, so this endpoint should automatically use this info if the creator is not specified by the frontend